packages/network/lib/src/bond_fire/base_bond_api_request.dart

```diff
@@ -21,7 +21,7 @@ class BaseBondApiRequest<T> {
 
   Map<String, String> _headers = <String, String>{};
   Map<String, dynamic> _queryParameters = <String, dynamic>{};
-  Map<String, dynamic>? _body;
```

Comment:
don't remove _body properly instead, create a new property called files and on bond fire execute method handle the files Map is not empty, create FormData